### ISSUE TITLE
KVMI-6 vs KVMI-7 vs MASTER

### ISSUE THREAD

**Issue by aghamir** [KVM-VMI/kvm#40]
Hi guys,
Can you explain what is the difference between these branches? I want to update my application which was written two years ago. Which branch should I work on?
Best wishes,

**Comment by Wenzel** [comment]
Hi @aghamir,

Thanks for your question, i will add a section to the documentation explaining the branches.

- master: main stable branch, currently holding kvmi-v6 but will move to kvmi-v7 soon
- kvmi-v6: tracks a working state of KVM-VMI for v6, currently the same as master.
- kvmi-v7: tracks a working state of KVM-VMI for v7

If you developed your application 2 years ago, i guess it was written based on the `nitro` system ? (Syscall interception).

This system has been deprecated in favor of Bitdefender's KVMi.
A backup branch exists as `nitro`.

I hope this helps clarifly the situation of the branches which I recognize is confusing.

I'm doing my best to offer a good level of support and write new documentation.

Cheers !

**Comment by aghamir** [comment]
Thanks a lot Mathieu. No I was built my application on the early `kvmi` release(1 or 2 months after you change the name of this repository if I remember correctly).
BTW, you helped me a lot.
Is KVMI-v7 stable now? I guess I will move to v7 too.

**Comment by Wenzel** [comment]
Yes, you should move to kvmi-v7, i shiuld post a detailed versioning between v6 and v7

Happy hacking 👌
